### PR TITLE
Fix OVMF builds caused by BASECORE dependencies

### PR DESCRIPTION
**Pull request by kuqin12**
Fix OVMF builds caused by BASECORE dependencies.

### CODE REVIEW

**Comment by kuqin12** [comment]
@spbrogan Can we enable the OVMF build validation as well? They have not been validated but the build results have been reflected in the commit history, which has been broken for a while..

**Review by spbrogan — APPROVED** [review]

File: Platforms/OvmfPkg/OvmfPkgX64.fdf [hunk 1]
```diff
@@ -115,6 +115,7 @@ SET gUefiCpuPkgTokenSpaceGuid.PcdSevEsWorkAreaSize = gUefiOvmfPkgTokenSpaceGuid.
 ################################################################################
 
 [FV.SECFV]
+FvForceRebase      = FALSE
```
Comment (kuqin12) [inline Platforms/OvmfPkg/OvmfPkgX64.fdf:118]: @TaylorBeebe @spbrogan Is this right? If I do not add them here, the alignment flag from X64 tools def will make the GenFv puke with "PE image Section-Alignment and File-Alignment do not match" on the PEI and SEC modules.

Comment (TaylorBeebe) [reply Platforms/OvmfPkg/OvmfPkgX64.fdf:118]: I ran into this a few months ago when trying to align PEI modules and started digging into it but was stopped by Sean. I think the reason he gave was that we would never allow the space bloat of aligning PEI/SEC modules. However, wouldn't SEC and PEI modules be built with the IA32 flags?

Comment (kuqin12) [reply Platforms/OvmfPkg/OvmfPkgX64.fdf:118]: This specific platform build is entirely based on x64 flags, as indicated in the file name. I agree that PEI/SEC should not expand size, but in that case, we need to override the build options for SEC and PEI modules. Do you have any suggestions on how to do that?

Comment (kuqin12) [reply Platforms/OvmfPkg/OvmfPkgX64.fdf]: Well, reverted that as we do not need to build them anymore.

**Review by TaylorBeebe — APPROVED** [review]

**Pull request merged**